test/skate_web/controllers/detour_route_controller_test.exs

```diff
@@ -177,7 +177,24 @@ defmodule SkateWeb.DetourRouteControllerTest do
           coordinates: [%{"lat" => 1, "lon" => 100}, %{"lat" => 2, "lon" => 101}]
         )
 
-      assert json_response(conn, 500)
+      assert %{"error" => %{"type" => "unknown"}} = json_response(conn, 500)
+    end
+
+    @tag :authenticated
+    test "returns a 500-level response with type: :no_route when ORS reports point not found error",
```

Comment:
Swooping in at later-than-the-last-minute!

I appreciate this test because it makes it abundantly clear what the relationship between state-of-the-world and output is. 

It also exposes a question: Would this particular use case make more sense as a 4xx-level error, because it's a problem with user input rather than a problem with the server?